motion_predict/include/motion_predict/motion_predict.hpp

```diff
@@ -67,17 +68,19 @@ namespace cv{
     \param  ay acceleration noise along y-axis (m^2/s^4)
     \param  process_noise_max is the maximum process noise of the system
     \param  confidence_drop_rate rate of drop in confidence with time
+    \note twist.linear vector is expected to be in the map frame and used as the yaw for future prediction.
```

Comment:
This might be confusing since there is no twist or pose args for this function. Could we add a note to the `obj` comment, i.e. "\param  obj external object; expected in map frame" or similar?

Reply:
ditto